src/EditorFeatures/Core/LanguageServer/AlwaysActivateInProcLanguageClient.cs

```diff
@@ -69,28 +72,15 @@ public override ServerCapabilities GetCapabilities(ClientCapabilities clientCapa
 
         serverCapabilities.SupportsDiagnosticRequests = true;
         serverCapabilities.DiagnosticProvider ??= new();
+
+        // VS does not distinguish between document and workspace diagnostics, so we need to merge them.
+        var diagnosticSourceNames = _diagnosticSourceManager.GetSourceNames(true)
```

Comment:
named arguments for both `true` and `false`.  it's not apparant what `GetSourceNames(true)` means.

Reply:
Would this be better named as GetDignosticCategories?  'SourceName' doesn't mean much afaict.